assets/src/app/app.tsx

```diff
@@ -8,15 +8,15 @@ import { SessionProvider } from '~/app/providers/session';
 
 export const App = () => (
   <React.StrictMode>
-    <ProviderRootStore>
-      <ReactQueryProvider>
+    <ReactQueryProvider>
```

Comment:
You didn't notice the problem that the timeout request to get supervisors could go through 2 times because the landing store was created 2 times (constructor call).

I found out that if <React.StrictMode> is placed AFTER <ProviderRootStore> the problem went away, have you seen something like this?

Reply:
It's not fixed, it's react.strictmode logic. 1 way to fix it -> delete react strict mode